packages/main/src/plugin/image-registry.ts

```diff
@@ -94,6 +98,23 @@ export class ImageRegistry {
     }
   }
 
+  loadUserDefaultRegistryConfig(): void {
+    this.defaultRegistries = this.configurationRegistry.getConfiguration('registries').get('defaults', []);
```

Comment:
should we put `registries` and `defaults` in a const since we use it multiple times (and below too for `registry`).